README.md

```diff
@@ -7,3 +7,16 @@ Connector for BI](https://docs.mongodb.com/bi-connector/current/),
 please see the
 [mongodb/mongo-bi-connector-odbc-driver](https://github.com/mongodb/mongo-bi-connector-odbc-driver)
 repository.
+
+
+## Set up the ODBC driver
+1. If you are not on a Windows machine, install the [unixODBC](http://www.unixodbc.org/) driver manager. Windows users can utilize the built-in driver manager.
+2. Update the values of `Driver`, `Pwd`, `Server`, `User`, and `Database` in `setup/setupDSN.reg`. The value of `Driver` should be the absolute path of `mongo-odbc-driver/target/debug/mongoodbc.dll`.
+3. Open the registry editor and add a new key under `ODBC/ODBCINST.INI` called `ADL_ODBC`. Then, add two sub-entries to `ADL_ODBC`:
+    ````
+    Driver: <path to dll>
+    Setup: <path to dll>
+   ````
+4. Run `reg import "setup/setupDSN.reg"` to populate the registry editor with the correct subkeys, entries, and values. You can run `reg query "HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBCINST.INI\ADL_ODBC_DRIVER"` to determine if the command ran successfully.
```

Comment:
The reg query command and using ODBC Admin are both validation steps. Do you want to put them in the same section?